scripts/regression/test_cnbeta.py

```diff
@@ -29,23 +29,23 @@ def main():
     wing.taper           = wing.chords.tip / wing.chords.root
     wing.aspect_ratio    = wing.spans.projected**2/wing.areas.reference
     wing.symmetric       = True
-    wing.origin          = np.array([58.6,0.,3.6]) * Units.feet  
-    
-    reference = SUAVE.Structure.Container()
+    wing.origin          = np.array([58.6,0.,3.6]) * Units.feet
+
+    #reference = SUAVE.Structure.Container()
     vehicle.reference_area = wing.areas.reference
     vehicle.append_component(wing)
-    
+
     wing = SUAVE.Components.Wings.Wing()
     wing.tag = 'Vertical Stabilizer'
     vertical = SUAVE.Components.Wings.Wing()
     vertical.spans.exposed = 32.4   * Units.feet
-    vertical.chords.fuselage_intersect = 38.7 * Units.feet
+    vertical.chords.root   = 38.7 * Units.feet      # vertical.chords.fuselage_intersect
```

Comment:
It's fine. There was a change in vertical root chord definition (now defined as root of the physical stabilizer rather than the reference surface extended to the fuselage centerline).